### DISCUSSION TITLE
<class 'AttributeError'>

### DISCUSSION THREAD

**Discussion by kprklg** [netbox-community/netbox#16737] [General]
Good afternoon
When loading images of the front or rear of the device
https://github.com/netbox-community/netbox/discussions/13899
same problem

<class 'AttributeError'>

This FileResponse instance has no `content` attribute. Use `streaming_content` instead.

Python: 3.11.2
NetBox: 4.0.6
plugins: 
  netbox_acls: 1.6.1
  netbox_attachments: 5.0.1
  netbox_bgp: 0.13.2
  netbox_contextmenus: 1.4.5
  netbox_data_flows: 1.0.0
  netbox_dns: 1.0.4
  netbox_documents: 0.7.0
  netbox_floorplan: 0.4.0
  netbox_interface_synchronization: 4.0.3
  netbox_inventory: 2.0.1
  netbox_ipcalculator: 1.4.4
  netbox_lifecycle: 1.0.4
  netbox_qrcode: 0.0.13
  netbox_reorder_rack: 1.1.1
  netbox_topology_views: 4.0.0
  nextbox_ui_plugin: 0.15.0
  phonebox_plugin: v0.0.9

happened after update 4.0.6

**Comment by candlerb** [comment]
Search for "streaming_content" in the discussion groups, and you'll find that `netbox_contextmenus` is a broken plugin that gives exactly this error. https://github.com/netbox-community/netbox/discussions/16579#discussioncomment-9857180

**Comment by kprklg** [comment]
https://github.com/PieterL75/netbox_contextmenus/issues/7

Thank you
